projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -6,55 +6,55 @@
 
 ### Enhancements
 
-- Added the change setting feature in Social for the auto conversion feature. [#32712]
-- Adds support for newsletter categories. [#32666]
-- ai-chat block: Ability to chat with the site. [#32741]
-- AI Excerpt: add Diacard button to the panel. [#32890]
-- AI Excerpt: add `Accept` button. [#32889]
-- AI Excerpt: connect UI with data layer. [#32875]
-- AI Excerpt: disable Generate button. [#32924]
-- AI Excerpt: do not disable Genertate button when there is a request error. [#32927]
-- AI Excerpt: ensure to save the post before to ask a suggestion. [#32891]
-- AI Excerpt: pick and pass raw post content to the server side. [#32939]
-- AI Excerpt: re enable generate button when options change. [#32943]
-- AI Excerpt: show excerpt number of words. [#32881]
-- AI Excerpt: show notice when request fails. [#32917]
+- AI-chat block: added adility to chat with the site. [#32741]
+- AI Excerpt: added the Discard button to the panel. [#32890]
+- AI Excerpt: added the "Accept" button. [#32889]
```

Comment:
Looking at past changeelogs, seems we like to say "add" rather than "added".

Also, a typo. And I don't know whether quotes around the "Discard" and "Accept" button names is good or not, but we should do the same in both places.
```suggestion
- AI-chat block: add ability to chat with the site. [#32741]
- AI Excerpt: add the Discard button to the panel. [#32890]
- AI Excerpt: add the Accept button. [#32889]
```